CHANGELOG.md

```diff
@@ -4,47 +4,47 @@ NOTE:
 
 ## Table of contents
 
-- [r1.1 - rc]
+- **[r1.1 - rc](#r11---rc)**
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
 The below sections record the changes for each API version in each (pre-)release as follows:
 
-* for the first alpha or release-candidate API version, all changes since the release of the previous public API version
-* for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
-* for a public API version, the consolidated changes since the release of the previous public API version
-
+* for the first alpha or release-candidate API version, all changes since the
+release of the previous public API version
+* for subsequent alpha or release-candidate API versions, the delta with
+respect to the previous pre-release
+* for a public API version, the consolidated changes since the release of the
+previous public API version
 
+# r1.1 - rc
 ## Release Notes
 
 This release contains the definition and documentation of
-* Connectivity Insights API
-* Application Profiles API
+* Connectivity Insights API v0.4.0
+* Connectivity Insights Subscriptions API v0.4.0
+* Application Profiles API v0.3.0
 
 The API definition(s) are based on
 * Commonalities v0.4.0-rc.1
 * Identity and Consent Management v0.2.0-rc.1
 
-
 ## Connectivity Insights API v0.4.0-rc.1
 
-** 0.4.0-rc.1 is the first release-candidate version for connectivity insights**
-
-
-### Main changes
-- This version is primarily to align with the changes to Commonalities 0.4-rc.1 and Identity and Consent Management v0.2.0-rc.1.
+0.4.0-rc.1 is the first release of the API. The version is aligned with
+Commonalities 0.4.0-rc.2 and Identity and Consent Management v0.2.0-rc.2.
```

Comment:
"Commonalities 0.4.0-rc.2 and Identity and Consent Management v0.2.0-rc.2" should be "upgraded" in lines 25-26 (29-30)

Reply:
issue #94 created to track this.

Reply:
fix available in commit [630372b](https://github.com/camaraproject/ConnectivityInsights/pull/67/commits/630372b35ec23abd958e129a5c45e49e1876354a)